testing/docker/bookinfo.yaml

```diff
@@ -56,7 +53,7 @@ reviews-v2:
     - A8_CONTROLLER_URL=http://controller:8080
     - A8_CONTROLLER_POLL=5s
     - A8_REGISTRY_POLL=5s
-    - A8_HEALTHCHECKS=http://localhost:9080/health
+    - A8_PROXY_ADAPTER=envoy
   external_links:
```

Comment:
@GregHanson @ijsnellf GH blew this away, when you changed the proxy adapter to nginx. just fyi